src/applications/mhv-medications/util/helpers.js

```diff
@@ -301,3 +301,77 @@ export const pharmacyPhoneNumber = prescription => {
   }
   return null;
 };
+
+/**
+ * This function sanitizes the input how we want it displayed when
+ * receiving the HTML string from the Krames API
+ *
+ * @param {String} htmlString - HTML string from the Krames API
+ */
+export const sanitizeKramesHtmlStr = htmlString => {
+  const tempDiv = document.createElement('div');
+  tempDiv.innerHTML = htmlString;
+
+  if (tempDiv.querySelector('body')) {
```

Comment:
would it be possible to cover this `if` condition in a unit test?

Reply:
This was an issue that can't be addressed in the test although it is working